src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -1,166 +1,166 @@
-                password: Some(ref pwd),
-                ref db,
-            }) = test_file.parsed_options
-            {
-                client
-                    .drop_and_create_user(
-                        user,
-                        pwd.as_str(),
-                        &[],
-                        &[AuthMechanism::ScramSha1, AuthMechanism::ScramSha256],
-                        db.as_deref(),
-                    )
-                    .await
-                    .unwrap();
-            }
-
-            let mut options_with_tls = options.clone();
-            if requires_tls {
-                let tls_options = TlsOptions::builder()
-                    .allow_invalid_certificates(true)
-                    .build();
-                options_with_tls.tls = Some(Tls::Enabled(tls_options));
-            }
-
-            let client = Client::with_options(options_with_tls).unwrap();
-            client
-                .database("db")
-                .run_command(doc! { "ping" : 1 }, None)
-                .await
-                .unwrap();
-            let mut actual_hosts = client.get_hosts().await;
-
-            test_file.hosts.sort();
-            actual_hosts.sort();
-
-            assert_eq!(test_file.hosts, actual_hosts);
-        }
-
-        if let Some(ref mut resolved_options) = test_file.options {
-            // When an `authSource` is provided without any other authentication information, we do
-            // not keep track of it within a Credential. The options present in the spec tests
-            // expect the `authSource` be present regardless of whether a Credential should be
-            // created, so the value of the `authSource` is not asserted on to avoid this
-            // discrepancy.
-            assert_eq!(resolved_options.replica_set, options.repl_set_name);
-            assert_eq!(resolved_options.ssl, options.tls_options().is_some());
-        }
-
-        if let Some(parsed_options) = test_file.parsed_options {
-            let actual_options = options
-                .credential
-                .map(|cred| ParsedOptions {
-                    user: cred.username,
-                    password: cred.password,
-                    // In some spec tests, neither the `authSource` or `db` field are given, but in
-                    // order to pass all the auth and URI options tests, the driver populates the
-                    // credential's `source` field with "admin". To make it easier to assert here,
-                    // we only populate the makeshift options with the credential's source if the
-                    // JSON also specifies one of the database fields.
-                    db: parsed_options.db.as_ref().and(cred.source),
-                })
-                .unwrap_or_default();
-
-            assert_eq!(parsed_options, actual_options);
-        }
-    }
-
-    run_spec_test(&["initial-dns-seedlist-discovery"], run_test).await;
-}
+// use serde::Deserialize;
```

Comment:
I don't think we should commit the changes to comment out these tests

Reply:
Nope, just left them in here to get better evergreen results. It looks like the DNS issue may be fixed, so I'll revert this portion now.